code/extensions/GroupSubsites.php

```diff
@@ -36,59 +58,63 @@ public function requireDefaultRecords()
             DB::query('UPDATE "Group" SET "AccessAllSubsites" = 1 WHERE "SubsiteID" = 0');
 
             // Move the field out of the way so that this migration doesn't get executed again
-            DB::get_schema()->renameField('Group', 'SubsiteID', '_obsolete_SubsiteID');
+            DB::get_schema()->renameField(Group::class, 'SubsiteID', '_obsolete_SubsiteID');
```

Comment:
This will again need `/** @skipUpgrade */` to tell the upgrader to leave this alone.

If you get tired of this everywhere, just add it once to the class doc for each file. :) The upgrader will check the whole [abstract syntax tree](https://en.wikipedia.org/wiki/Abstract_syntax_tree) before making the substitution.